src/vstart.sh

```diff
@@ -227,6 +231,14 @@ case $1 in
     --short )
 	    short=1
 	    ;;
+    --crimson )
+        ceph_osd=crimson-osd
+        ;;
+    --smp )
```

Comment:
i created a separated commit with your Signed-off-by in #26697 for this improvement.